app/plans/get.controller.js

```diff
@@ -1,27 +1,31 @@
+const { logger } = require('../../common/logging/logger')
 const { getSentencePlanSummary } = require('../../common/data/sentencePlanningApi')
 const { isEmptyObject } = require('../../common/utils/util')
 
-const hasActivePlan = plans => {
-  if (isEmptyObject(plans)) return false
-  let activePlanPresent = false
-  plans.forEach(plan => {
-    if (!plan.completedDate) {
-      activePlanPresent = true
-    }
-  })
-  return activePlanPresent
+const getPlan = plans => {
+  if (isEmptyObject(plans)) return {}
+  return plans.find(({ completedDate }) => completedDate === '' || completedDate === undefined) || {}
 }
 
 const sentencePlanSummary = async ({ params: { id }, session: { 'x-auth-token': token } }, res) => {
-  const plans = await getSentencePlanSummary(id, token)
-  const renderInfo = {}
-  if (!plans || isEmptyObject(plans)) {
-    renderInfo.activePlan = false
-  } else {
-    renderInfo.activePlan = hasActivePlan(plans)
+  try {
+    const plans = await getSentencePlanSummary(id, token)
+    const renderInfo = {}
+    renderInfo.individualId = id
+    renderInfo.currentPlan = getPlan(plans)
+    if (isEmptyObject(renderInfo.currentPlan)) {
+      renderInfo.planType = 'none'
+    } else if (renderInfo.currentPlan.isDraft) {
+      renderInfo.planType = 'draft'
+    } else {
+      renderInfo.planType = 'active'
+    }
```

Comment:
Not in any way a merge blocker, more a minor style point here I guess; 
renderInfo.planType 'active' never gets checked, its the else scenario in the render.
So you could just do:
```
if (isEmptyObject(renderInfo.currentPlan)) {
      renderInfo.planType = 'none'
    } else if (renderInfo.currentPlan.isDraft) {
      renderInfo.planType = 'draft'
    }
```
Or if you wanted just always set it to `active` as the default and then re-assign if either of the two other conditions are true?